aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -11,6 +11,8 @@ spec aptos_framework::account {
         aborts_if !system_addresses::is_aptos_framework_address(aptos_addr);
         aborts_if exists<OriginatingAddress>(aptos_addr);
         ensures exists<OriginatingAddress>(aptos_addr);
+        // TODO: an compiler error is occur when using table::new()
+        //ensures global<OriginatingAddress>(aptos_addr).address_map == table::new();
```

Comment:
This error is expected. `table::new` cannot be used in spec. Unfortunately, there seems no good way to specify about `global<OriginatingAddress>(aptos_addr).address_map`.

Reply:
yeah, guess we have to left it for now